### DISCUSSION TITLE
［娱乐］knock summer车队招聘

### DISCUSSION THREAD

**Discussion by Prance36** [share121/inter-knot#4948] [其他]
![招聘](https://github.com/user-attachments/assets/175e85a7-e729-4eac-b7f1-e728c4dfe5ae)
在今年的比赛当中， **_stick knock summer_** 车队表现不佳，频繁出现各种问题，如车队的工程用邦布换胎系统故障，车辆动力单元故障，车辆空气动力学套件质量问题等。
在 **_Auto sport_** 接手之后，车队决定在全新艾利都范围内招兵买马，以解决当前问题。
如果你有赛车维修及调试经验或工程用邦布维修调试经验，均可与我们联系，或者如果你有合适的人选，也可将其推荐给我们。

**Comment by zzz20100624** [comment]
我是25-1号车手🤡🤡

**Comment by LucasLH44** [comment]
我从kick sauber离职，可以加入吗🤡
